packages/@local/package-chores/tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+  "compilerOptions": {
+    "allowSyntheticDefaultImports": true,
+    "allowJs": true,
```

Comment:
```suggestion
```

Reply:
Good suggestions! I did not bother with tsconfig details as I was going to extract it into a separate package anyway. I'll address your comments as part of [Asana task](https://app.asana.com/0/0/1202210072243248/f) (internal)